new_buildpack.go

```diff
@@ -0,0 +1,104 @@
+package pack
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/buildpacks/pack/internal/build"
+	"github.com/buildpacks/pack/internal/dist"
+	"github.com/buildpacks/pack/internal/style"
+)
+
+var (
+	bashBinBuild = `
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+layers_dir="$1"
+env_dir="$2/env"
+plan_path="$3"
+
+exit 0
+`
+	bashBinDetect = `
+#!/usr/bin/env bash
+
+exit 0
+`
+)
+
+type NewBuildpackOptions struct {
+	// The base directory to generate assets
+	Path string
+
+	// The ID of the output buildpack artifact.
+	ID string
+
+	// The stacks this buildpack will work with
+	Stacks []dist.Stack
+}
+
+func (c *Client) NewBuildpack(ctx context.Context, opts NewBuildpackOptions) error {
+	buildpackTOML := dist.BuildpackDescriptor{
+		API:    build.SupportedPlatformAPIVersions.Latest(),
+		Stacks: opts.Stacks,
+		Info: dist.BuildpackInfo{
+			ID:      opts.ID,
+			Version: "0.0.0",
+		},
+	}
+
+	f, err := os.Create(filepath.Join(opts.Path, "buildpack.toml"))
+	if err != nil {
+		return err
+	}
+	if err := toml.NewEncoder(f).Encode(buildpackTOML); err != nil {
+		return err
+	}
+	defer f.Close()
+	c.logger.Infof("    %s  buildpack.toml", style.Key("create"))
```

Comment:
```suggestion
	c.logger.Infof("    %s  buildpack.toml", style.Symbol("new"))
```
Symbol wraps Key for situations when color isn't enabled.

Reply:
Actually, do you think it makes more sense to highlight creating the file, or which file we created? Should we be highlight create/new, or buildpack.toml (and all the other files)?